frontend/src/components/Label/Labels.tsx

```diff
@@ -8,18 +8,18 @@ const SHOW_MORE_TRESHOLD = 2;
 
 interface LabelsProps {
   expanded?: boolean;
-  type?: string;
   labels?: { [key: string]: string };
   tooltipMessage?: string;
+  type?: string;
 }
 
 const linkStyle = kialiStyle({
   padding: '0 0.25rem',
   fontSize: '0.8rem'
 });
 
-const infoStyle = kialiStyle({
-  marginLeft: '0.25rem',
+const labelInfoStyle = kialiStyle({
+  marginLeft: '0.125rem',
```

Comment:
Because the label already introduces a margin-right of `0.25rem` (see image below), if we want the space between the label and the info icon to be `0.375rem`, we need to add a margin-left of `0.125rem` to the info icon.

![image](https://github.com/user-attachments/assets/46f4374b-06a2-4c02-ba95-1ee1b1b216fb)

Reply:
ah, OK, I see now that you commented on this above. Thanks.